packages/commonwealth/client/scripts/models/Comment.ts

```diff
@@ -66,7 +65,7 @@ export class Comment<T extends IUniqueId> {
     content_url,
   }) {
     const versionHistory = CommentVersionHistories;
-    this.communityId = community_id;
+    this.communityId = Address?.community_id;
```

Comment:
This is weird that community_id is undefined on the Comment model. Bug might be fixed by it but probably would be good to investigate it further why is that happening

Reply:
@masvelio I thought this same thing after I had pushed this fix. @mzparacha you have any thoughts or insight on this?

Reply:
we can use `comment.thread.community_id` and possibly revert this @Israellund

Reply:
Thanks @mzparacha